recirq/qml_lfe/sample_analysis.ipynb

```diff
@@ -0,0 +1,1451 @@
+      "text/plain": [
+       "<Figure size 201.6x216 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
```

Comment:
This is a long code block with no explanation.  I would add comments or some other explanation so that it is more clear what's going on here.